### PR TITLE
Fix radio button glitch

### PR DESCRIPTION
**Pull request by gavinorland**
Rounding these numbers does fix https://github.com/UKHomeOffice/govuk-react/issues/350 in the Storybook - but I don't know if they need to be that long fractional value for some special reason.

**Checklist**:

* [x] Documentation
* [x] Tests
* [x] Ready to be merged

### CODE REVIEW

**Comment by gavinorland** [comment]
Oops - will fix these tests...

File: components/radio/src/index.js [hunk 1]
```diff
@@ -64,20 +65,20 @@ const LabelText = styled('span')({
     position: 'absolute',
     top: 0,
     left: 0,
-    width: '2.10526em',
-    height: '2.10526em',
+    width: '2em',
```
[the remark this answers is not in the document]
Comment (gavinorland) [reply components/radio/src/index.js]: Yes, these edits do make the elements visibly slightly smaller. Wasn't quite sure on the scope for flexibility with units here. I just tried with pixels, but fractional values are needed to match that size in `em`s and the glitch comes back. With points, as you suggested, it is doable and I have a very good match - even a slightly truer circle.

**Comment by gavinorland** [comment]
This is at Steve's suggestion - thought he might have reason to prefer points - you could check with him. I'm fine with pixels instead.

**Comment by gavinorland** [comment]
Actually (this was three weeks ago!), if you open up the "Outdated" feedback above, there is a bit of discussion on this.

**Comment by Loque-** [comment]
Steve is on holiday, I did read the comments above but it didn't explain why pt over px.

**Comment by gavinorland** [comment]
Sorry, I thought it did. Steve suggested pixels. He may have his own reason for preferring these - I don't know - but as you say he's on holiday at the moment so we can't check. I also added:

>  I just tried with pixels, but fractional values are needed to match that size in ems and the glitch comes back. With points, as you suggested, it is doable and I have a very good match - even a slightly truer circle.

Feel free to have a play around with it and see how you get on if you'd like to use pixels :)

**Comment by Loque-** [comment]
I've adjusted it to use px in the name of consistency, it seems to render fine - if anyone would like to validate that before it goes in, could be good, but it's a pretty minimal and quick change to adjust. Hope that's helpful!

**Pull request merged**